client/src/analytics/events.ts

```diff
@@ -50,7 +50,8 @@ export enum EVENTS {
   EXPLORER_GENE_INFO_BUTTON_CLICKED = "EXPLORER_GENE_INFO_BUTTON_CLICKED",
   WMG_CLICK_NAV = "WMG_CLICK_NAV",
   COLLECTIONS_CLICK_NAV = "COLLECTIONS_CLICK_NAV",
-  CENSUS_CLICK_NAV = "CENSUS_CLICK_NAV",
+  CENSUS_DOCUMENTATION_CLICK_NAV = "CENSUS_DOCUMENTATION_CLICK_NAV",
+  CENSUS_DIRECTORY_CLICK_NAV = "CENSUS_DIRECTORY_CLICK_NAV",
```

Comment:
I assume she knows about the addition of these two events and safe to assume these are in Plausible too?